app/scripts/components/table/list-view.js

```diff
@@ -176,9 +171,14 @@ class List extends React.Component {
       tableSortProps,
       bulkActions,
       rowId,
-      list
+      list,
+      list: {
```

Comment:
You can de-structure the same property twice? In this case allowing you to also de-structure nested properties. I don't know that I think it's clearer, but I don't have any issue with it

Reply:
Yep, no need for a separate destructuring assignment. It's arguably slightly more clear because of proximity. With the previous separate destructuring assignment, the reader must search for where `list` is declared. With this syntax, it appears on the line above, so there's no need to "hunt" for the declaration. Yes, very minor, but I found it helpful.